src/main/java/nz/cri/gns/NZSHM22/opensha/griddedSeismicity/NZSHM22_GridSourceGenerator.java

```diff
@@ -70,7 +70,7 @@
 	 */
 	public NZSHM22_GridSourceGenerator(NZSHM22_InversionFaultSystemSolution ifss) {
 		branch = ifss.getLogicTreeBranch();
-		srcSpatialPDF = branch.getValue(NZSHM22_SpatialSeisPDF.class).getPDF();
+		srcSpatialPDF = branch.getValue(NZSHM22_SpatialSeisPDF.class).getPDF(new NewZealandRegions.NZ_TEST_GRIDDED()); // FIXME later
```

Comment:
@voj what's the FIXME later for?

Reply:
We need to figure out what the regions mean for hazard. I reckon we'll do that when we get there.